search/feature_offset_match.hpp

```diff
@@ -329,18 +329,10 @@ void MatchPostcodesInTrie(TokenSlice const & slice, trie::Iterator<ValueList> co
   impl::OffsetIntersector<Filter, Value> intersector(filter);
   for (size_t i = 0; i < slice.Size(); ++i)
   {
-    if (slice.IsPrefix(i))
-    {
-      std::vector<PrefixDFAModifier<UniStringDFA>> dfas;
-      slice.Get(i).ForEach([&dfas](UniString const & s) { dfas.emplace_back(UniStringDFA(s)); });
-      MatchInTrie(dfas, TrieRootPrefix<ValueList>(*postcodesRoot, edge), intersector);
-    }
-    else
-    {
-      std::vector<UniStringDFA> dfas;
-      slice.Get(i).ForEach([&dfas](UniString const & s) { dfas.emplace_back(s); });
-      MatchInTrie(dfas, TrieRootPrefix<ValueList>(*postcodesRoot, edge), intersector);
-    }
+    // Full match required even for prefix token.
```

Comment:
Please add the reasoning behind the change to this comment.

Reply:
done